src/app/auth/signup.tsx

```diff
@@ -44,18 +87,78 @@ function SignUpScreen() {
         <Input
           label="Password"
           leftIcon={{ type: 'font-awesome', name: 'lock' }}
-          onChangeText={text => setPassword(text)}
+          onChangeText={text => checkPassword(text)}
           value={password}
           secureTextEntry
           placeholder="Password"
           autoCapitalize="none"
         />
+      </View>
+      <View>
+        {password !== '' && (
+          <View style={globalStyles.passwordComplexity}>
+            <Icon type={hasUppercase ? 'green_check' : 'red_x'} />
+            <Text
+              style={
+                hasUppercase
+                  ? { color: colors.textGreen }
+                  : { color: colors.textRed }
+              }
+            >
+              At least 1 uppercase letter.
```

Comment:
Nit pick, but to match the designs exactly, let's remove periods from the ends of all of the complexity text lines